provider/aws/config.go

```diff
@@ -80,11 +94,25 @@ func CreateV2Configs(cfg *externaldns.Config) map[string]awsv2.Config {
 }
 
 func newV2Config(awsConfig AWSSessionConfig) (awsv2.Config, error) {
+	tlsConfig, err := tlsutils.NewTLSConfig(awsConfig.TLSCertPath, awsConfig.TLSCertKeyPath, awsConfig.TLSCAFilePath, "", awsConfig.TLSSkipVerify,
+		tls.VersionTLS13)
+
+	if err != nil {
+		return awsv2.Config{}, fmt.Errorf("instantiating TLS config: %w", err)
+	}
+
+	httpClient := &http.Client{}
+	if tlsConfig != nil {
```

Comment:
What is your plan to test this? How to prove this is actually working?

Reply:
I wrote a unit test and need to test it.. The plan is to create a CA and then use the CA to create a cert and use that for two way TLS.